### PR TITLE
feat(suite-native): use filled icons for focused items in tab bar

### PR DESCRIPTION
**Pull request by yanascz**
https://github.com/user-attachments/assets/6ae8fedd-f24f-4a3e-8c0b-fd5b70689da3

### CODE REVIEW

File: suite-native/app/src/navigation/routes.ts [hunk 1]
```diff
@@ -21,11 +23,13 @@ const receiveStack = enhanceTabOption({
     routeName: AppTabsRoutes.ReceiveStack,
     label: 'Receive',
     iconName: 'arrowLineDown',
+    focusedIconName: 'arrowLineDownFilled',
```
Comment (Nodonisko) [inline suite-native/app/src/navigation/routes.ts]: That focused arrow icon looks really ugly IMHO I would keep it same as unfocused

Comment (yanascz) [reply suite-native/app/src/navigation/routes.ts]: I kinda agree. Let's see what has @shenkys to say.

Comment (yanascz) [reply suite-native/app/src/navigation/routes.ts]: Fixed.

**Review by vytick — APPROVED** [review]
Looks good, nice improvement 🚀

**Comment by STew790** [comment]
**QA OK**
Icons are correctly filled when clicked on.
image

**Info**
25.1.2 https://github.com/trezor/trezor-suite/commit/4c73755ef3de25c4ae1a69ae05abcce85b1d55ab

**Pull request merged**